<ui version="4.0" >
 <class>frm_sysinfo</class>
 <widget class="QWidget" name="frm_sysinfo" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>系统信息</string>
  </property>
  <property name="styleSheet" >
   <string notr="true" >font: 12pt "WenQuanYi Zen Hei";
background-color: rgb(112, 200, 11);</string>
  </property>
  <widget class="QPushButton" name="pbtn_ok" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>240</y>
     <width>100</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text" >
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>180</y>
     <width>90</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>版本号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>130</y>
     <width>90</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>型号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>70</y>
     <width>90</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>设备名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>70</y>
     <width>330</width>
     <height>34</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>120</y>
     <width>330</width>
     <height>34</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>180</y>
     <width>330</width>
     <height>34</height>
    </rect>
   </property>
   <property name="dragEnabled" >
    <bool>false</bool>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
